2.4/root/usr/bin/run-mongod

```diff
@@ -47,9 +47,7 @@ function cleanup() {
   fi
   echo "=> Shutting down MongoDB server ..."
   if [ -f "${MONGODB_PID_FILE}" ]; then
-    kill -2 $(cat ${MONGODB_PID_FILE})
-  else
-    pkill -2 mongod
+    mongod --shutdown $mongo_common_args
```

Comment:
so let's just always call it then, and do an || true to avoid an error resulting in us exiting.  or do a set +e/set -e